packages/fast-usdc/src/update-noble-ica.core.js

```diff
@@ -38,7 +38,7 @@ const makeRatio = (numerator, numeratorBrand, denominator = 100n) =>
 const updateFeeConfig = (
   feeConfigPre,
   usdcBrand,
-  variableRate = makeRatio(5n, usdcBrand, 1000n),
+  variableRate = makeRatio(1n, usdcBrand, 1000n),
```

Comment:
this could be:

```suggestion
  variableRate = makeRatio(10n, usdcBrand, BASIS_POINTS),
```

with

```js
const BASIS_POINTS = 100n * 100n;
```

above somewhere.